samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -39,6 +39,7 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
 	imply MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT
 
 choice MCUMGR_TRANSPORT_BT_PERM
+	bool
```

Comment:
Why this is added here?

Reply:
Pretty sure it threw KConfig warnings, but I see that they have been updated in general, can try without.